scripts/install

```diff
@@ -309,7 +285,7 @@ fi
 #=================================================
 ynh_script_progression --message="Integrating service in YunoHost..."
 
-yunohost service add $service_name --description "Creates a Wi-Fi access point"
+yunohost service add $service_name --description "Creates a Wi-Fi access point" --test_status "systemctl is-active hostapd" --needs_exposed_ports 547 67
```

Comment:
Re-reading this - I understand you probably added --needs_exposed_ports because the linter told you to haha - but in the specific case of this apps, these ports are only relevant for the local network I'm guessing ... but the thing is that "needs_exposed_ports" will automatically add a check in the diagnosis that the port is publicly exposed just like port 80 ... I havent really thought about this use case ... Maybe I could use the fact that we have --no-upnp in the "yunohost firewall allow" to interpret this as a local-network-only port.

Reply:
We really need something like `// eslint-disable-line rule-name`